script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -689,99 +812,115 @@ The sovereigns of the Dead Apostles. Beings considered by the Executors as the h
+"I had Karius send her back. She's got a briefing she's gotta get to anyway.
 }
 [sha:40ad5f565906e035fee67b1a8d316c5633db03a1]{
 -- Page 847, Offset 13128.
 -- 埋葬機関の方は話にならねえが、あの女は分かりやすい。ニンジンぶらさげりゃあ死ぬまで走るタマだ。
 // I'm not sure what 走るタマ is. Is it like 球が走る in baseball or 霊?
-It's no use talking about the Burial Agency, but that girl is easy to understand. Dangle a carrot in front of her, and she'll run until she dies.
+It's no use talking to the one from the Burial Agency, but that girl's easy to read.
+ Dangle a carrot in front of her, and she'll run until she dies.
 }
 [sha:62db4961b0dec13fac3951e6e471ed232b5ee5f6]{
 -- Page 847, Offset 13129.
 -- んで、そっちはどうだ？」
-'nyways, how did it go on your side?"
+So, how'd it go on your end?"
 }
 [sha:e242c2c62fe562d59e5ece0434969569e771aadc]{
 -- Page 848, Offset 13130. Mods: @n.
 -- 「いやあ、何も残っていませんわ。
-"Weell, there's nothing left.
+"Weell, I didn't find squat.
 }
 [sha:1408b4ddb6362e9335efa4cf74fd2683a821da3e]{
 -- Page 848, Offset 13132.
 -- 死徒にとっての聖典、吸血鬼の<魂跡|こんせき>ってヤツ？　死体も残ってないですし、あの代行者が回収しちまったみたいですね」
-Things like Scriptures about Dead Apostles or the vampire's Soul Remains? Not even the corpses are left, it looks like that Executor collected everything."
+No Scriptures related to the Dead Apostles, no trace that vampire was ever there.
+ There aren't even any corpses left lying around, seems like that Executor went and collected everything."
 }
 [sha:f163a2138cf3e5af85f24e8170464d9c4bb99ccd]{
 -- Page 848, Offset 13133.
 -- 「チッ、またかよあのアマ。戦利品はきっちり上納するもんだろうが。組織の上下関係舐めてんのか」
-"Tch, that fucking bitch again. Spoils of war should always be left as tribute. She's making fun of our organization's hierarchy, huh?"
+// 上納 is specifically payment to govt, so I guess here tithes to make it ~churchy~
+"%{i}Tch%{/i}, I can't believe that bitch did it again.
+ The spoils of war should always be left as a tithe.
+ Is she fucking making fun of our organization's hierarchy, huh?"
 }
 [sha:99882a66c8445515ef4650c199d99a4655d1846c]{
 -- Page 849, Offset 13134.
 -- 「いやあ、そんなワケのわからない<戦利品|モノ>押しつけられても困りますけどねぇ。
-// It would be best to rephrase it to delete the pronouns
-"Weell, I wouldn't know what to do, if such incomprehensible things were loaded off to me.
+"I meeannn, I wouldn't have a clue what to do with half of that stuff it ended up being pawned off on me.
 }
 [sha:06fe8741a9216d109759e76fe1a5e3bcb00c5dda]{
 -- Page 849, Offset 13135.
 -- 餅は餅屋、化け物は化け物殺しにでいいじゃないッスか」
-Every man to his trades, so isn't it fine to leave monsters to the monster-killer?"
+// Not sure if there's a pithier one of this in EN.
+// 'Every man to his trade' has more of a 'stay in your lane' vibe that I think isn't present here as much
+// Alternative would be to localize mochi (We leave bread to bakers, so doesn't it...)
+// Bread analogy also amuses me because of Ciel baking
+You want good results you go to a specialist, so doesn't it make sense to leave monsters to the monster-killer?"
```

Comment:
localising to bread sounds good to me

Reply:
i wouldn't mind that localization but this line is fine as-is imo